src/main/java/be/ugent/rml/extractor/ReferenceExtractor.java

```diff
@@ -22,13 +25,7 @@ public ReferenceExtractor(String reference) {
 
     @Override
     public List<Object> extract(Record record) {
-        String temp = this.reference;
-
-        if (ignoreDoubleQuotes && temp.startsWith("\"") && temp.endsWith("\"")) {
-            temp = temp.substring(1, temp.length() - 1);
-        }
```

Comment:
Ah you moved this problem to the constructor so it only happens once?

Reply:
Yeah, same as with to UpperCase.